styles.css

```diff
@@ -443,6 +456,114 @@ li .list-item #prg-lang {
   font-size: 15px;
 }
 
+.contact-me {
+  background-image: url("images/image-for-contact\ me.png");
+  width: 360px;
+  height: 779px;
+  top: 5410px;
+  display: flex;
+  flex-direction: column;
+  margin-top: 0;
+}
+
+.form-header {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 130px;
+  margin-bottom: 30px;
+}
+
+.form-heading {
+  color: #fff;
+  width: 801px;
+  height: 56px;
+  top: 4734px;
+  margin-top: 60px;
+  font-family: "Poppins", sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: center;
+}
+
+.pg-contact {
+  width: 327px;
+  height: 137px;
+  top: 4802px;
+  left: 24px;
+  font-family: "Poppins", sans-serif;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #fff;
+}
+
+.form-info {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+input,
+label,
+textarea {
+  width: 327px;
+  height: 48px;
+  border-radius: 5px;
+  border: 1px;
+  margin-left: 20px;
+  color: black;
+  padding: 5px;
+}
+
+label {
+  color: #fff;
+}
+
+textarea {
+  resize: none;
+  width: 327px;
+  height: 145px;
+  border-radius: 8px;
+  border: 1px;
+  color: gray;
+}
+
+.button-form {
+  width: 142px;
+  height: 48px;
+  top: 5317px;
```

Comment:
- [ ] Kindly consider adding a `hover` effect to your button to match that of the `Figma` design. It would enhance the user experience and provide visual feedback. You can achieve this by utilizing CSS and applying a `hover` pseudo-class selector to the `button` element. Consider adding appropriate CSS properties such as background color and text color effects to create the desired `hover` effect. See below for more details of the colors to use:

<img width="762" alt="Screenshot 2023-07-04 at 16 07 07" src="https://github.com/umarahgul/Portfolio/assets/107192076/e041b310-a7cc-4124-9352-b6a6c8c50d6f">

Reply:
@Strangeal . If, I add a blue border then, the button border entirely disappears on 'hover'.  This doesn't look nice from UX. So, I want to add a white border, which will enhance the UX.  Does this negatively affect my project?

Reply:
@umarahgul No it doesn't negatively affect your project, Having the white border is good. I just checked.